README.md

```diff
@@ -35,6 +35,7 @@ $ csv-to-influxdb --help
                            (default timestamp)
   --timestamp-format, -tf  Timestamp format used to parse all timestamp
                            records (default 2006-01-02 15:04:05)
+                           use "unix" for parse unix timestamp time values
```

Comment:
Thanks for the PR! The CLI help text is defined near the top of main.go, please add this there